### PR TITLE
30-min hero-style guide for sync w/ a JS client & cloud app

### PR DESCRIPTION
**Pull request by damienomurchu**
__not ready to be merged - just wip__

renamed from 'beginnings of a sync-getting-started doc' to '30-min hero-style guide for sync w/ a JS client & cloud app'

Doc to provide a complete guide/ set of steps to illustrate with example apps how to incorporate fh-sync into JS client app and server apps.

### CODE REVIEW

File: hero-sync.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+# Using Sync for your Client & Server data
+
+A set of client and server apps to demonstrate sync may be found here:
+
+* [Sample feedhenry sync-client app][sync-client-template]
+* [Sample feedhenry sync-cloud app][sync-cloud-template]
+
+
+[sync-client-template]: https://github.com/feedhenry-templates/sync-app
```
Comment (david-martin) [inline hero-sync.md]: I don't think this sync app is maintained. It isn't included in the RHMAP platform templates https://github.com/feedhenry/fh-template-apps/blob/master/global.json

Did you mean to reference https://github.com/feedhenry-templates/sync-cordova-app ?

**Comment by david-martin** [comment]
@damienomurchu I'm not sure about merging this given the examples use fh-mbaas-api & fh-js-sdk, which require RHMAP to run on. It is possible to get them working without RHMAP, but it may be better to hold off until the sync client & server are extracted into their own library, and we create standalone templates for them then.
Does that make sense?

**Comment by damienomurchu** [comment]
@david-martin don't worry - nowhere near ready for merging yet - just & a skeleton of a doc that will change a lot yet

**Comment by wtrocki** [comment]
@damienomurchu IMHO this provide zero value over the actual sync documentation which has *exactly the same content* and this content will be monitored and updated. Duplication it's not the best way to promote opensource stuff. 

What we can do is to help on finalizing work on fh-sync and fh-sync-js around documentation (basically to make it publicly available) Happy to help you guys on this.

**Comment by damienomurchu** [comment]
@wtrocki purpose of this guide is not to duplicate existing documentation but to supplement it with a quick-start doc for developers looking to use sync immediately & point to existing apps/ docs as appropriate. 

Outstanding fh-sync & fh-sync-js issues also being looked at

**Comment by wtrocki** [comment]
@damienomurchu actual sync documentation has quick start section.

**Comment by damienomurchu** [comment]
@wtrocki link? couldn't find a clean client-sync app example, all examples mired in cordova/ angular/ ionic code structures

**Comment by wtrocki** [comment]
@damienomurchu - Sync documentation is now merged. You can add examples and snippets there as we going to move this docs to feedhenry anyway.

**Comment by damienomurchu** [comment]
@wtrocki yes, seen that. @austincunningham & I just working on completing this doc & will then house it where best. Link on feedhenry.org will just be a pointer to where the doc is housed with the rest of the sync docs

**Comment by secondsun** [comment]
Closing in favor of hosting on the project docs for now.

**Pull request closed without merging**